common/workunit/workunit.hpp

```diff
@@ -1768,6 +1768,8 @@ extern WORKUNIT_API void runK8sJob(const char *componentName, const char *wuid,
 
 // return the k8s external host and port for serviceName
 extern WORKUNIT_API std::pair<std::string, unsigned> getExternalService(const char *serviceName);
+
 #endif
+extern WORKUNIT_API std::vector<std::vector<std::string>> getPodNodes(const char *selector);
```

Comment:
trivial: Should be inside the #endif above.

Reply:
fixed + squashed